sdk/monitor/Azure.Monitor.Ingestion/src/autorest.md

```diff
@@ -0,0 +1,29 @@
+# Azure SDK Code Generation for Data Plane
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+input-file:
+- https://github.com/Azure/azure-rest-api-specs/blob/c6d856b6ca7f079e96ed8680bd867b8d6c197669/specification/monitor/data-plane/ingestion/preview/2021-11-01-preview/DataCollectionRules.json
+namespace: Azure.Monitor.Ingestion
+security: AADToken
+security-scopes: https://monitor.azure.com//.default
+```
+
+### todo: rename streamName
```

Comment:
Nit - consider filing bugs on open TODOs that we're not addressing today and including the bug numbers in comments like this.  It makes it easier to cross reference in the future.